ui/components/app/permission-cell/permission-cell.js

```diff
@@ -53,6 +53,11 @@ const PermissionCell = ({
     iconBackgroundColor = Color.backgroundAlternative;
   }
 
+  let permissionIcon = avatarIcon;
+  if (typeof avatarIcon !== 'string' && avatarIcon?.props?.iconName) {
+    permissionIcon = avatarIcon.props.iconName;
```

Comment:
Why is this now needed?

Reply:
Can we leave a comment to fix this later? We should probably make some changes in the permission-connect UI to fix this instead